src/mlpack/tests/split_data_test.cpp

```diff
@@ -362,10 +357,9 @@ TEST_CASE("SplitDataResultField", "[SplitDataTest]")
   mat matA(2, 10);
   mat matB(2, 10);
 
-  size_t count = 0; // Counter for unique sequential values.
-  matA.imbue([&count]() { return ++count; });
-  matB.imbue([&count]() { return ++count; });
-
+  matA = linspace(0, matA.n_elem - 1);
+  matA = linspace(matA.n_elem, matA.n_elem + matB.n_elem - 1);
```

Comment:
Should this be setting `matB` instead of `matA`?

```suggestion
  matB = linspace(matA.n_elem, matA.n_elem + matB.n_elem - 1);
```

Reply:
Yes you are right, probably a regexp error, funny enough, it was not caught by  tests